library/src/lib/panel/panel-grid/panel-grid.component.ts

```diff
@@ -15,7 +15,7 @@ export class PanelGridComponent extends AbstractFdNgxClass {
     @Input()
     col: number;
 
-    /** Whether the grid shoul have a gap. */
+    /** Whether the grid should have a gap. */
     @Input()
     @HostBinding('class.fd-panel-grid--nogap')
```

Comment:
is there a point in keeping this line if we're just adding the class in setProperties?

Reply:
No, there is no point in adding classes in HostBinding.